### ISSUE TITLE
Please update galaxy role ansible-zookeeper

### ISSUE THREAD

**Issue by patsevanton** [andrewrothstein/ansible-zookeeper#5]
Hello!
Thanks for role ansible-zookeeper

```
fatal: [172.26.10.11]: FAILED! => {
    "changed": false, 
    "dest": "/tmp/apache-zookeeper-3.5.6-bin.tar.gz", 
    "response": "HTTP Error 404: Not Found", 
    "state": "absent", 
    "status_code": 404, 
    "url": "http://apache.org/dist/zookeeper/zookeeper-3.5.6/apache-zookeeper-3.5.6-bin.tar.gz"
}

MSG:

Request failed
```

**Comment by andrewrothstein** [comment]
latest version of the role published on the galaxy has the checksums for zookeeper 3.5.7 as well as 3.6.0. the default is 3.6.0.

**Comment by patsevanton** [comment]
Simple Playbook install_zookeeper_cluster.yml
```
---
- name: Install ZooKeeper cluster
  hosts: zookeeper
  roles:
    - role: andrewrothstein.zookeeper
    - role: andrewrothstein.zookeeper-cluster
```

Output
```

fatal: [zookeeper-2]: FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "attributes": null,
            "backup": null,
            "checksum": "sha256:db24700e453f2ad4b6b789d553fe828ccceef3977a7e6b36389580ec92397a7e",
            "client_cert": null,
            "client_key": null,
            "content": null,
            "delimiter": null,
            "dest": "/tmp/apache-zookeeper-3.5.6-bin.tar.gz",
            "directory_mode": null,
            "follow": false,
            "force": false,
            "force_basic_auth": false,
            "group": null,
            "headers": null,
            "http_agent": "ansible-httpget",
            "mode": 420,
            "owner": null,
            "regexp": null,
            "remote_src": null,
            "selevel": null,
            "serole": null,
            "setype": null,
            "seuser": null,
            "sha256sum": "",
            "src": null,
            "timeout": 600,
            "tmp_dest": null,
            "unsafe_writes": null,
            "url": "http://apache.org/dist/zookeeper/zookeeper-3.5.6/apache-zookeeper-3.5.6-bin.tar.gz",
            "url_password": null,
            "url_username": null,
            "use_proxy": true,
            "validate_certs": true
        }
    }
}

MSG:

Failed to connect to downloads.apache.org at port 443: [Errno 101] Network is unreachable
```

**Comment by patsevanton** [comment]
https://github.com/andrewrothstein/ansible-zookeeper-cluster/issues/9

**Comment by andrewrothstein** [comment]
published v3.0.12 which defaults to zookeeper:3.6.2

**Comment by andrewrothstein** [comment]
actually v3.0.12 was broken. just tested and published a fix as v3.1.0
